Pages/Shared/_Layout.cshtml.css

```diff
@@ -1,7 +1,7 @@
 ﻿/* Please see documentation at https://learn.microsoft.com/aspnet/core/client-side/bundling-and-minification
 for details on configuring this project to bundle and minify static web assets. */
 
-a.navbar-brand {
+a.navbar-brand-link {
```

Comment:
Clarifies the purpose of the class, adhering to semantic and descriptive naming conventions.

Reply:
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions